src/components/app-configure/CreateApp.vue

```diff
@@ -105,44 +105,46 @@ async function handleCreateApp() {
     <VCard variant="popup" class="create-app-modal-card">
       <h2 class="create-app-title">Create New App</h2>
       <VSeperator />
-      <VStack direction="column" gap="1rem">
-        <label class="app-name-label">Enter App Name</label>
-        <VTextField
-          v-model="appName"
-          class="app-name-input"
-          :message-type="hasAppNameError ? 'error' : ''"
-          message="App Name cannot be empty"
-        />
-      </VStack>
-      <VStack direction="column" gap="1rem" align="start">
-        <VStack gap="0.5rem">
-          <label class="app-name-label">Choose Region</label>
-          <VTooltip
-            title="Arcana Store uses physical storage nodes that are logically grouped by
+      <form @submit.prevent="handleCreateApp">
+        <VStack direction="column" gap="1rem">
+          <label class="app-name-label">Enter App Name</label>
+          <VTextField
+            v-model.trim="appName"
+            class="app-name-input"
+            :message-type="hasAppNameError ? 'error' : ''"
+            message="App Name cannot be empty"
+          />
```

Comment:
Associate the label with the input field with `for` and `id` attributes. Please re-check that this is true for all forms input fields added in this PR.

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/128/commits/ce585351c74236bbc80e5b138373f586c935d4a9